giotto/mapper/visualization.py

```diff
@@ -15,36 +15,56 @@
 from sklearn.base import clone
 
 from .utils._logging import OutputWidgetHandler
-from .utils.visualization import (_get_colorscale_buttons, _get_colorscales,
-                                  _get_column_color_buttons, _get_node_size,
-                                  _get_node_text, get_node_summary,
-                                  set_node_sizeref)
+from .utils.visualization import (_get_column_color_buttons, _get_node_colors,
+                                  _get_node_size, _get_node_text,
+                                  _is_array_or_dataframe, set_node_sizeref)
 
 
-def create_network_2d(graph, data, node_pos, node_color,
-                      columns_to_color=None, plotly_kwargs=None):
+def create_static_network(pipeline, data, layout='kamada_kawai', layout_dim=2,
+                          color_variable=None, node_color_statistic=np.mean,
+                          color_by_columns_dropdown=True, plotly_kwargs=None):
     """
     Parameters
     ----------
-    graph : igraph.Graph
-        The nerve of the refined pullback cover. Nodes correspond to cluster
-        sets, and an edge exists between two nodes if they share at least one
-        point in common.
-
-    data : ndarray of shape (n_samples, n_features)
-        The point cloud data used to generate the nerve.
-
-    node_pos : igraph.Graph.layout or ndarray of shape (n_nodes, n_dims)
-        Encodes the layout of the graph according to a layout algorithm or
-        pre-defined array of coordinates in an n-dimensional space.
-
-    node_color : ndarray of shape (n_nodes,)
-        The numerical values to color each node of the graph by.
-
-    columns_to_color : dict, optional, default: ``None``
-        Key-value pairs (column_name, column_index) to specify which columns of
-        :attr:`data` to color the graph by. Generates a dropdown widget to
-        select the columns to color by.
+    pipeline : :class:`MapperPipeline` object
+        Mapper pipeline to act on to data.
+
+    data : array-like of shape (n_samples, n_features)
+        Data used to generate the Mapper graph. Can be a pandas dataframe.
+
+    layout : None, str or callable, optional, default: ``'kamada-kawai'``
+        Layout algorithm for the graph. Can be any accepted value for the
+        ``layout`` parameter in the :meth:`layout` method of
+        :class:`igraph.Graph`. [1]_
+
+    layout_dim : int, default: ``2``
+        The number of dimensions for the layout. Can be 2 or 3.
+
+    color_variable : column index or name, or list of such, \
```

Comment:
Perhaps, to avoid the current long line describing available types, this entry can be rewritten as follows:
- Remove the long list of object types and change it to 'object or None, optional, default: ``None``'
- Detail the various possibilities in a bullet point list (careful with newlines and indents, for reference you can check the docstring for `PairwiseDistance`)

Reply:
Agreed. You'll see my attempt with the next push.